exercises/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,19 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+var greeting = "HELLO ";
```

Comment:
I see how you approached this one. Rember you need two functions. so your greeting is supposed to be a function and not a variable. 
see the instruction "Your program should include a function that spells their name in uppercase, and a function that creates a shouty greeting."

let me know if you need help working out the solution.